rules/prefer-modern-dom-apis.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const checkForReplaceChildOrInsertBefore = (context, node) => {
+	const getArgumentName = args => {
+		if (args.type === 'Identifier') {
+			return args.name;
+		}
+
+		return null;
+	};
+
+	const forbiddenIdentifierNames = new Map([
+		['replaceChild', 'replaceWith'],
+		['insertBefore', 'before']
+	]);
```

Comment:
Move this to a constant variable outside of `checkForReplaceChildOrInsertBefore`.

Reply:
Do you mean the global scope?
Why would I move it outside when it is only used in `checkForReplaceChildOrInsertBefore`?
The more scoped the better imo

Reply:
We recently introduced a rule that will enforce this behavior. See the discussion here: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/230.

The short version is that there isn't any significant benefit to scoping the function and a couple minor benefits to code readability, maintainability and performance if you move it out.